integration/data/test_restore_to_file.py

```diff
@@ -140,7 +142,15 @@ def restore_to_file_with_backing_file_test(backing_dev, backup_target):
     os.remove(output_qcow2_path)
     assert not os.path.exists(output_qcow2_path)
 
+    launcher.shutdown_engine_frontend()
```

Comment:
Can we put those wrappers around `cmd.snapshot_revert` into a common function as well?